src/domains/transaction/pages/SendTransfer/FormStep.tsx

```diff
@@ -97,8 +105,9 @@ export const FormStep = ({
 	const handleSelectSender = (address: any) => {
 		setValue("senderAddress", address, { shouldDirty: true, shouldValidate: false });
 
-		const senderWallet = profile.wallets().findByAddressWithNetwork(address, network.id());
+		const newSenderWallet = profile.wallets().findByAddressWithNetwork(address, network.id());
 		const isFullyRestoredAndSynced = senderWallet?.hasBeenFullyRestored() && senderWallet?.hasSyncedWithNetwork();
+		setSenderWallet(newSenderWallet);
```

Comment:
```suggestion
		const newSenderWallet = profile.wallets().findByAddressWithNetwork(address, network.id());
		const isFullyRestoredAndSynced = newSenderWallet?.hasBeenFullyRestored() && newSenderWallet?.hasSyncedWithNetwork();
```
we need to check the new selected address to see if it's fully synchronized, right now it's checking the old one

Reply:
Good catch, thanks